iocore/net/UnixNet.cc

```diff
@@ -465,6 +433,9 @@ NetHandler::mainNetEvent(int event, Event *e)
     epd = (EventIO *)get_ev_data(pd, x);
     if (epd->type == EVENTIO_READWRITE_VC) {
       vc = epd->data.vc;
+      if (cop_list.in(vc)) {
+        cop_list.remove(vc);
+      }
```

Comment:
Thank you for the clarification.  I am good with the changes.  Do you have any numbers around this change and the benefits?

There will be changes to the scheduler and the frequency maybe more than that in the future.  I don't see that as an issue.